docs/docs/development.md

```diff
@@ -5,10 +5,10 @@ Assessment Workspace Collaborative project.  To begin you should have the
 following applications installed on your local development system:
 
 - [Git](https://git-scm.com/)
-- [Python](https://www.python.org/) == 3.9
+- [Python](https://www.python.org/) == 3.10
```

Comment:
the current python version we have deployed is 3.9; can we keep this one back on 3.9 for now? there's a PR we'll merge in next quarter to move to 3.11, see https://github.com/shapiromatron/hawc/pull/769

Reply:
just tested with python 3.9 and it still works! going to send in a new pull request with the change